swizzle/provision.py

```diff
@@ -98,6 +99,10 @@ def generate_inventory(node_state={}):
     with open(temp_file, 'w') as fh:
         parser.write(fh)
 
+    print "Running with inventory:\n"
+    parser.write(sys.stdout)
```

Comment:
do you need `sys.stdout.flush()` somewhere around here?

Reply:
There should be no reason that we need to force the flush of the buffer here.  Additionally, this is only meant to be informational - there is no runtime impact (other than some lines on a console).